.travis.yml

```diff
@@ -0,0 +1,41 @@
+language: php
+
+php:
+  - 7.1
+  - 7.2
+
+# faster builds on new travis setup not using sudo
+sudo: false
+
+# cache vendor dirs
+cache:
+  directories:
+    - $HOME/.composer/cache
+
+install:
+  - travis_retry composer self-update && composer --version
+  - export PATH="$HOME/.composer/vendor/bin:$PATH"
+  - travis_retry composer install --dev --prefer-dist --no-interaction
+# install php extensions
+  - |
+    pear config-set preferred_state beta
+    printf "\n" | pecl install imagick
+    # gmagick is not installed on travis currently
+    #printf "\n" | pecl install gmagick
+# setup database:
+  - mysql -e 'CREATE DATABASE yii2test;';
+# setup application:
+  - |
+    php install.php init/all --config=tests/travis/install-config.php --interactive=0
+    sed -i "s/'cookieValidationKey' => ''/'cookieValidationKey' => 'testkey'/" config/web.php
```

Comment:
Not needed. It's a console.

Reply:
I'm going to fix it tomorrow, and I thought at first, make sure it works, and then remove the useless stuff, which should be more efficient, and more people can be involved 🙁